src/components/UserInfo.js

```diff
@@ -12,8 +13,9 @@ export default class UserInfo {
     return userInfo;
   }
 
-  setUserInfo(data) {
-    this._userNameElement.textContent = data.title;
-    this._userJobElement.textContent = data.description;
+  setUserInfo({ name, about, avatar }) {
+    this._userNameElement.textContent = name;
+    this._userJobElement.textContent = about;
+    this._userAvatarElement.src = avatar;
```

Comment:
Needs correction: It's needed to create a separate setter for avatar `setAvatar` and use it when needed to update the avatar, because user info and avatar are controlled by different forms.

When initializing user info you can use bit methods:

```js
userInfo.setUserInfo({ title, description })
userInfo.setAvatar(avatar)
```